docs/src/main/asciidoc/deploying-native-executable-openshift.adoc

```diff
@@ -0,0 +1,218 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id="deploy-openshift-native-executable"]
+= Deploy a Quarkus native application on OpenShift
+include::_attributes.adoc[]
+:diataxis-type: howto
+:categories: cloud,native
+
+Deploy native Quarkus applications to OpenShift Container Platform by using the Docker build strategy enabled by the Quarkus OpenShift extension.
+
+Quarkus supports the `quarkus-container-image-docker` extension for building container images.
+The extension uses the Docker binary and generated Dockerfiles in the `src/main/docker` directory to perform Docker builds.
```

Comment:
The extension (meaning the quarkus-container-image-openshift) is not using the docker binary. It is uploading the native binary and the dockerfiles to openshift and openshift performs the build.

In fact this could all work without the user having docker installed. I am saying `could` because in some cases the `docker` binary may be used in order to create the native binary itself (as correctly described by the paragraph below).

Reply:
Would the following update be correct instead:  "The `quarkus-container-image-openshift` extension uploads a native binary and Dockerfiles to OpenShift and OpenShift performs the Docker builds."